Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h

```diff
@@ -0,0 +1,123 @@
+      int ITSnClusters = itsTrk.getNClusters();
+      float ITSchi2 = itsTrk.getChi2();
+      float itsChi2NCl = ITSnClusters != 0 ? ITSchi2 / (float)ITSnClusters : 0;
+      uint8_t itsClusterMap = itsTrk.getPattern();
+      if (itsTrk.getPt() >= mMinPt && itsTrk.getPt() <= mMaxPt &&
+          itsTrk.getEta() >= mMinEta && itsTrk.getEta() <= mMaxEta &&
+          ITSnClusters >= mMinNClustersITS &&
+          itsChi2NCl <= mMaxChi2PerClusterITS &&
+          TrackMethods::FulfillsITSHitRequirements(itsClusterMap, mRequiredITSHits)) {
+        return true;
+      } else {
+        return false;
+      }
+    } else if (contributorsGID[GIndex::Source::TPC].isIndexSet()) {
+      const auto& tpcTrk = data.getTPCTrack(contributorsGID[GID::TPC]);
+      uint8_t tpcNClsShared, tpcNClsFound, tpcNClsCrossed, tpcNClsFindable, tpcChi2NCl;
+      // tpcNClsFindable = tpcTrk.getNClusters();
+      // tpcChi2NCl = tpcTrk.getNClusters() ? tpcTrk.getChi2() / tpcTrk.getNClusters() : 0;
+      // o2::TrackMethods::countTPCClusters(tpcTrk, data.getTPCTracksClusterRefs(), data.clusterShMapTPC, data.getTPCClusters(), tpcNClsShared, tpcNClsFound, tpcNClsCrossed);
+      // double tpcCrossedRowsOverFindableCls = tpcNClsCrossed / tpcNClsFindable;
+      math_utils::Point3D<float> v{};
+      std::array<float, 2> dca;
+      if (tpcTrk.getPt() < mPtTPCCut ||
+          std::abs(tpcTrk.getEta()) > mEtaTPCCut ||
+          tpcTrk.getNClusters() < mNTPCClustersCut ||
+          (!(const_cast<o2::tpc::TrackTPC&>(tpcTrk).propagateParamToDCA(v, mBz, &dca, mDCACut)) ||
+           std::abs(dca[0]) > mDCACutY)) {
+        return false;
+      } else {
+        return true;
+      }
+    } else if (contributorsGID[GIndex::Source::TRD].isIndexSet()) {
+      const auto& trdTrk = data.getTrack<o2::trd::TrackTRD>(contributorsGID[GID::TRD]);
+    } else if (contributorsGID[GIndex::Source::TOF].isIndexSet()) {
+      const auto& tofMatch = data.getTOFMatch(trackIndex);
+    } else if (contributorsGID[GIndex::Source::TPCTRDTOF].isIndexSet()) {
+      trk = data.getTrack<o2::track::TrackParCov>(data.getTPCTRDTOFMatches()[trackIndex].getTrackRef());
+    } else if (contributorsGID[GIndex::Source::ITSTPCTRDTOF].isIndexSet()) {
+      trk = data.getTrack<o2::track::TrackParCov>(data.getITSTPCTRDTOFMatches()[trackIndex].getTrackRef()); // ITSTPCTRDTOF is ITSTPCTRD + TOF cluster
+    } else if (contributorsGID[GIndex::Source::ITSTPCTOF].isIndexSet()) {
+      trk = data.getTrack<o2::track::TrackParCov>(data.getTOFMatch(trackIndex).getTrackRef()); // ITSTPCTOF is ITSTPC + TOF cluster
+    } else {                                                                                   // for the rest, get the track directly
+      trk = data.getTrack<o2::track::TrackParCov>(trackIndex);
+    }
+    if (trk.getPt() >= mMinPt && trk.getPt() <= mMaxPt &&
+        trk.getEta() >= mMinEta && trk.getEta() <= mMaxEta) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
... substitute all this by:
```
if (isBarrelTrack) {
  trk  = data.getTrackParam(trackIndex);
  return (trk.getPt() >= mMinPt && trk.getPt() <= mMaxPt &&  trk.getEta() >= mMinEta && trk.getEta() <= mMaxEta);
```

Reply:
I added this as a condition which returns true or false